Source/WSTagView.swift

```diff
@@ -166,8 +219,13 @@ open class WSTagView: UIView, UITextInputTraits {
 
     open override var intrinsicContentSize: CGSize {
         let labelIntrinsicSize = textLabel.intrinsicContentSize
-        return CGSize(width: labelIntrinsicSize.width + layoutMargins.left + layoutMargins.right,
-                      height: labelIntrinsicSize.height + layoutMargins.top + layoutMargins.bottom)
+        let imageSize = isImageHidden ? CGSize(width: 0, height: 0) : self.imageSize
```

Comment:
sidenote: the CG objects usually have a `.zero` property. So you could use `CGSize.zero` or `.zero` if Swift could infer the type.